### PR TITLE
add an extra format token to the primary data string so that JRuby do…

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
…esn't break

for: https://github.com/rails-api/active_model_serializers/pull/1004

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
@bf4, what do you think of this solution?

I can handle squashing and merging into rails-api/ams, if you are busy :-)

File: lib/active_model_serializers/json_pointer.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ module JsonPointer
 
     POINTERS = {
       attribute:    '/data/attributes/%s'.freeze,
-      primary_data: '/data'.freeze
+      primary_data: '/data%s'.freeze
```
Comment (bf4) [inline lib/active_model_serializers/json_pointer.rb:7]: Interesting.  Is this a JRuby bug?

**Comment by bf4** [comment]
Failure is in https://travis-ci.org/rails-api/active_model_serializers/jobs/111938627

File: lib/active_model_serializers/json_pointer.rb (hunk 1 above)
Comment (NullVoxPopuli) [inline lib/active_model_serializers/json_pointer.rb:7]: I believe jruby doesn't handle nil correctly, yeah

Comment (NullVoxPopuli) [inline lib/active_model_serializers/json_pointer.rb:7]: Nil value and no % variable*

**Comment by NullVoxPopuli** [comment]
Thaatt failure is from 9 days ago? Unless I misread it

**Comment by NullVoxPopuli** [comment]
Travis passes for this pr

File: lib/active_model_serializers/json_pointer.rb (hunk 1 above)
Comment (bf4) [inline lib/active_model_serializers/json_pointer.rb:7]: Would you mind linking to an open issue or creating one on https://github.com/jruby/jruby for the greater good? :)

**Comment by bf4** [comment]
@NullVoxPopuli I just meant that's the failure this PR is responding to

**Comment by NullVoxPopuli** [comment]
yes :-)

**Pull request merged**